scripts/output.sh

```diff
@@ -0,0 +1,9 @@
+set -e
+
+help='This script is run with the format  ./output.sh <target stack name> <cloudformation output variable name>'
```

Comment:
Surprised this is entirely new as a file. iirc in the other apps we updated an existing file. Do you know why this one would be different?